lib/src/module.dart

```diff
@@ -741,6 +767,64 @@ abstract class Module {
     return inOutPort;
   }
 
+  /// Registers a signal as an [inOut] to this [Module] and returns an inOut
```

Comment:
nit: second inOut -- do you want to bracket it, or define it as input/output

Reply:
thanks fixed here and in multiple places